lascli/parser/transitions.py

```diff
@@ -135,6 +141,12 @@ def create_transitions_parser(subparsers):
     update_execution_parser.add_argument('status', choices=['succeeded', 'failed', 'rejected', 'retry'])
     update_execution_parser.add_argument('--output_path', '-o')
     update_execution_parser.add_argument('--error_path', '-e')
+    update_execution_parser.add_argument('--start-time')
```

Comment:
I think we should make sending start time easier if possible. It is cumbersome typing in an ISO date string. Maybe use datetime somehow?

Reply:
I don't see when people will actually use this unless they have the exact time...

Reply:
I don't really think our users will use this feature in the CLI, so I suggest we leave it like this until we get a pushback.